code/modules/reagents/reagents/reagents_medical.dm

```diff
@@ -82,6 +82,25 @@
 	T.add_toxinlevel(5)
 	T.add_planthealth(50)
 
+/datum/reagent/panacea
+	name = "Panacea"
+	id = PANACEA
+	description = "A variant of Adminordrazine that has been subjected to medical sciences to make it incredibly potent. It's magic, stolen from the gods."
+	reagent_state = REAGENT_STATE_LIQUID
+	color = "#EECE19" //rgb: 238, 206, 25
+	density = ARBITRARILY_LARGE_NUMBER
+	specheatcap = ARBITRARILY_LARGE_NUMBER
```

Comment:
These two unironically make it more attractive as an engineering chem than a medicine, the infinite TEG.